client/web/api.go

```diff
@@ -20,16 +20,12 @@ type api struct {
 // which protects the handler using gorilla csrf.
 func (a *api) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("X-CSRF-Token", csrf.Token(r))
-	user, err := authorize(w, r)
-	if err != nil {
-		return
-	}
 	path := strings.TrimPrefix(r.URL.Path, "/api")
 	switch path {
 	case "/data":
 		switch r.Method {
 		case httpm.GET:
-			a.s.serveGetNodeDataJSON(w, r, user)
+			a.s.serveGetNodeDataJSON(w, r)
```

Comment:
Oh! I forgot to add the metrics counter here, mind adding?
```
s.lc.IncrementCounter(context.Background(), "web_client_page_load", 1)
```

Reply:
is page_load the right counter to increment here?  That would cause two counts, one for the HTML page, and then one for the data, wouldn't it?

Reply:
(going ahead and submitting without... we can add it later if this is indeed the right counter)